client/src/components/post/MainPost/MainPost.js

```diff
@@ -0,0 +1,33 @@
+
+import PostFooter from "../PostFooter/PostFooter.vue";
+export default {
+  name: 'MainPost',
+  components: {
+    PostFooter
+  },
+  props: {
+    header: String,
```

Comment:
Let's rename variables for header and body to postHeader and bodyHeader
Also the same for img -> imgHeader

It is for the compatibility issues